### PR TITLE
Merging #26074

### PR DESCRIPTION
**Pull request by nikitamikhaylov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):

- Not for changelog (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Original PR #26074

### CODE REVIEW

**Comment by nikitamikhaylov** [comment]
@Mergifyio update

File: src/AggregateFunctions/AggregateFunctionNothing.h [hunk 1]
```diff
@@ -46,7 +48,7 @@ class AggregateFunctionNothing final : public IAggregateFunctionHelper<Aggregate
 
     size_t sizeOfData() const override
     {
-        return 0;
+        return 1;
```
Comment (alexey-milovidov) [inline src/AggregateFunctions/AggregateFunctionNothing.h:51]: This looks unneeded. Not necessary to waste space in memory.

**Comment by nikitamikhaylov** [comment]
@alexey-milovidov Which data type should return `AggregateFunctionNothing`?  Better to return `DataTypeNothing`, but we can't create it

**Comment by alexey-milovidov** [comment]
Is it possible for it to return DataTypeNothing, and exception will be thrown in attempt to actually get the value?

**Comment by alexey-milovidov** [comment]
Looks totally alright.

**Comment by nikitamikhaylov** [comment]
Thank you!

**Pull request merged**